includes/admin/class-evf-admin-assets.php

```diff
@@ -178,6 +178,8 @@ public function admin_scripts() {
 					'i18n_field_meta_key_error'    => esc_html__( 'Please enter in meta key with alphanumeric characters, dashes and underscores.', 'everest-forms' ),
 					'i18n_field_min_value_greater' => esc_html__( 'Minimum value is greater than Maximum value.', 'everest-forms' ),
 					'i18n_field_max_value_smaller' => esc_html__( 'Maximum value is smaller than Minimum value.', 'everest-forms' ),
+					'ajax_import_nonce'            => wp_create_nonce( 'process-import-ajax-nonce' ),
+					'ajax_url'                     => admin_url( 'admin-ajax.php', 'relative' ),
```

Comment:
Since this is naked params so make sure this are placed first and `i18n_<strings>` params are placed afterwards.

Reply:
The pointed codes are moved appropriately to the top.